react/src/components/molecules/ListingTable/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+
+import Collapse from '../../animations/Collapse';
+
+import './style.css';
+
+class ListingTableItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
```

Comment:
I'm a big fan of using arrow functions for things like this so you don't have to bind in the constructor.

Reply:
Completed this, however it is throwing an error in the lint.